packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -24,35 +28,23 @@ export default {
 		size: { default: null, type: String },
 		type: { default: null, type: String },
 	},
-	data() {
+	setup(props, { emit, slots }) {
+		const hasSlot = computed(() => Boolean(slots.default))
 		return {
-			isHover: false,
+			handleClick: (event) => emit(KottiButton.Event.CLICK, event),
+			hasSlot,
+			mainClasses: computed(() => ({
+				'kt-button': true,
+				[props.type]: true,
```

Comment:
this can translate to a `null` key, as in
```
mainClasses = { 
//... other key value pairs ,

null: true 
}
```
in theory, wouldn't make a difference, but i hate it :(